packages/eslint-config/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@penumbra-zone/eslint-config",
+  "version": "1.0.0",
```

Comment:
Renamed this and also removed private. This way, external consumers (like our registry) can make use of this.